app/controllers/comments_controller.rb

```diff
@@ -14,9 +14,10 @@ def create
     )
     @new_comment.post_id = @post.id
     if @new_comment.save
-      redirect_to "/users/#{@post.author_id}/posts/#{@post.id}", notice: 'Success!'
+      redirect_to "/users/#{@post.author_id}/posts/#{@post.id}", flash: { alert: 'Your comment is saved' }
     else
-      render :new, alert: 'Error occured!'
+      flash.now[:error] = 'Could not save comment'
+      render action: 'new'
     end
```

Comment:
- [ ] I didn't receive a flash/notice message for a successful and not successful path. The flash/notice message is supposed to be rendered in the view but because you didn't configure it yet in the layout, the messages were not rendered in the view. Kindly add the code snippet I attached below in the `app/views/layouts/application.html.erb` before `yield` so that the flash/notice messages can be rendered in the views.

```ruby
    <% if notice %>
      <p><%= notice %></p>
    <% end %>
    <% if alert %>
      <p><%= alert %></p>
    <% end %>
```